tests/e2e/cluster_downgrade_test.go

```diff
@@ -298,14 +380,23 @@ func getMembersAndKeys(t *testing.T, cc *e2e.EtcdctlV3) (*clientv3.MemberListRes
 	return members, kvs
 }
 
-func generateIdenticalVersions(clusterSize int, ver *semver.Version) []*version.Versions {
+func generateIdenticalVersions(t *testing.T, clusterSize int, ver *semver.Version) []*version.Versions {
 	ret := make([]*version.Versions, clusterSize)
 
+	// storage version string is non-empty starting from 3.6.0
+	ver3_6_0, err := semver.NewVersion("3.6.0")
```

Comment:
can you reuse V3_6? https://github.com/etcd-io/etcd/blob/f30cbaac111aa01a310fe75360c759cdd4d9cd14/api/version/version.go#L44

Reply:
Ahh yes that would be better! Thanks!